src/xcode/ENA/ENA/Source/Developer Menu/Features/DMDeviceTimeCheck/DMDeviceTimeCheckViewModel.swift

```diff
@@ -0,0 +1,56 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+#if !RELEASE
+
+import Foundation
+import UIKit
+
+final class DMDeviceTimeCheckViewModel {
+
+	// MARK: - Init
+
+	init(store: Store) {
+		self.store = store
+	}
+
+	// MARK: - Overrides
+
+	// MARK: - Public
+
+	// MARK: - Internal
+
+	var itemsCount: Int {
+		return menuItems.allCases.count
+	}
+
+	func cellViewModel(for indexPath: IndexPath) -> DMSwitchCellViewModel {
+		guard let item = menuItems(rawValue: indexPath.row) else {
+			fatalError("failed to create cellViewModel")
+		}
+		switch item {
+
+		case .killDeviceTimeCheck:
+			return DMSwitchCellViewModel(
+				labelText: "Disable device time check",
+				isEnabled: { [store] in
+					store.dmKillDeviceTimeCheck ?? false
```

Comment:
Will the button be disabled if the store.dmKillDeviceTimeCheck is nil? I think the store.dmKillDeviceTimeCheck is nil initially and the button will never be enabled? Or did I miss something?

Reply:
Maybe it makes sense to put default "false" into the store property?

Reply:
The `isEnabled` refers to the state of the Switch, is it switched on or ist it switched off.

Reply:
Ok.. then the name is misleading. Should be "isOn"

Reply:
Done ✅